azure-pipelines-official.yml

```diff
@@ -84,26 +84,9 @@ jobs:
       sourcePath: '$(Build.SourcesDirectory)\artifacts\OptProf\$(BuildConfiguration)\Data'
       toLowerCase: false
       usePat: false
-    displayName: 'OPTPROF - Publish to Artifact Services - ProfilingInputs'
+    displayName: 'OptProf - Publish to Artifact Services - ProfilingInputs'
     condition: succeeded()
 
-  # Publish OptProf generated JSON files as a build artifact. This allows for easy inspection from
-  # a build execution.
-  - task: PublishBuildArtifacts@1
-    displayName: Publish OptProf Data Files
-    inputs:
-      PathtoPublish: '$(Build.SourcesDirectory)\artifacts\OptProf\$(BuildConfiguration)\Data'
-      ArtifactName: 'OptProf Data Files'
-    condition: succeeded()
```

Comment:
This task should still be present.

Reply:
It is missing from the master branch. Should it be there as well?

Reply:
There is a task for publishing optprof
```
  # Publish OptProf configuration files
  - task: ms-vscs-artifact.build-tasks.artifactDropTask-1.artifactDropTask@0
    inputs:
      dropServiceURI: 'https://devdiv.artifacts.visualstudio.com'
      buildNumber: 'ProfilingInputs/DevDiv/$(Build.Repository.Name)/$(Build.SourceBranchName)/$(Build.BuildId)'
      sourcePath: '$(Build.SourcesDirectory)\artifacts\OptProf\$(BuildConfiguration)\Data'
      toLowerCase: false
      usePat: false
    displayName: 'OptProf - Publish to Artifact Services - ProfilingInputs'
    condition: succeeded()
```

Reply:
It's missing from the master branch because I checked the change in to master-vs-deps originally. I haven't had a chance to port back to master yet.

Reply:
@jaredpar Sorry about that. I've restored the publish task.